README.md

```diff
@@ -5,7 +5,10 @@ open-vcdiff is an encoder and decoder for the VCDIFF format, as described in
 and Compression Data Format.
 
 You will need to first synchronize gflags and gtest by running
-`git submodule init --update --recursive`.
+`git submodule init --update --recursive`. Or if you have system installed
```

Comment:
I think this line will conflict with master, could you rebase? This was wrong apparently.